src/validations/address.validations.ts

```diff
@@ -0,0 +1,9 @@
+import { object, string } from 'yup'
+
+export const createAddressValidator = object().shape({
+  country: string().required(),
+  address: string().required(),
+  city: string().required(),
```

Comment:
It is an id the services/domain/address.service.ts , why is it a string here?

Reply:
We receive it from the frontend as string